aiida_pseudo/cli/show.py

```diff
@@ -17,10 +18,15 @@ def cmd_show(family, raw):
     """Show details of pseudo potential family."""
     from tabulate import tabulate
 
-    rows = [[pseudo.element, pseudo.filename, pseudo.md5] for pseudo in family.nodes]
-    headers = ['Element', 'Pseudo', 'MD5']
+    if isinstance(family, RecommendedCutoffMixin):
+        headers = ['Element', 'Pseudo', 'MD5', 'Wavefunction', 'Charge density']
```

Comment:
Should we also add the units to the cutoff headers here? For now it'd just be `eV`, but once we add support for other units these can be extracted from the family (e.g. using your suggested `get_recommended_cutoff_unit` method).

Reply:
Yes, good point. I was thinking of adding this when we address the issue of storing the unit explicitly in the extras, but might already hardcode it here that it is eV. I was already working on the stringency and it is almost done. Will push changes soon